modules/va_facilities/README.v1.yml

```diff
@@ -37,6 +37,18 @@ info:
     - The list of available services in the `services` element is only applicable to VA health and
       benefits facilities.
 
+    ### Mobile Facilities
```

Comment:
Why is there a README.yml and a README.v1.yml? Do we need both?

Reply:
We do need both. They provide the content for different versions of the API documentation that you can select in the dropdown here: https://developer.va.gov/explore/facilities/docs/facilities